src/main/java/com/force/i18n/grammar/AbstractLanguageDeclension.java

```diff
@@ -462,7 +461,7 @@ public final LanguagePluralRules getPluralRules() {
      * Often times, complex declension have simple forms of one thing or another.
      * This is the "simple" form types for which there is only one possible declension
      */
-    public static enum SimpleModifierForm implements AdjectiveForm, ArticleForm {
```

Comment:
removed redundant `static` modifier for inner-enum class ([java:S2786](https://rules.sonarsource.com/java/RSPEC-2786))